shell/platform/embedder/embedder.cc

```diff
@@ -606,8 +620,8 @@ FlutterEngineResult FlutterEngineInitialize(size_t version,
   }
 
   if (!IsRendererValid(config)) {
-    FML_LOG(WARNING) << "Invalid renderer config.";
-    return LOG_EMBEDDER_ERROR(kInvalidArguments);
+    return LOG_EMBEDDER_ERROR(kInvalidArguments,
+                              "The renderer configuration was not valid.");
```

Comment:
Desperate grasp at avoiding a line-wrap since github doesn't show an 80-col marker: would `s/not valid/invalid/` get you to 80 cols by any chance?

Reply:
Nope. But I updated it anyway.